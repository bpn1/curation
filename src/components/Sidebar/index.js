export {default as default} from './sidebar';
