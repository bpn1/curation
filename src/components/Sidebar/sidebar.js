import React, {Component} from 'react';
import Drawer from 'material-ui/Drawer';

class Sidebar extends Component {

  render() {

    const cstyle = {
      backgroundColor: '#EEEEEE',
      position: '',
      borderRight: '1px solid #CDCDCD',
      width: '205px',
    };

    const style = {
      height: 'inherit'
    };

    return (
      <Drawer open={this.props.showSideNav} zDepth={this.props.zDepth} style={style} containerStyle={cstyle}>
        {this.props.children}
      </Drawer>
    );
  }
}

export default Sidebar
