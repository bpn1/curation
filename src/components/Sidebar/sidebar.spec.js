import React from 'react'
import Sidebar from './sidebar'
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'

test('Sidebar snapshot test', () =>{
  const component = shallow(<MuiThemeProvider><Sidebar>Testo</Sidebar></MuiThemeProvider>);
  expect(component).toMatchSnapshot()
});
