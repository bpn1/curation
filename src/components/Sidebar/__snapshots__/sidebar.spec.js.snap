exports[`test Sidebar snapshot test 1`] = `
<Sidebar>
  Testo
</Sidebar>
`;
